extensions/collection-assets/examples/example-esm.json

```diff
@@ -0,0 +1,97 @@
+{
+  "stac_version": "0.9.0",
+  "stac_extensions": [
+    "collection-assets",
+    "https://github.com/NCAR/esm-collection-spec/tree/v0.2.0/schema.json"
+  ],
+  "id": "pangeo-cmip6",
+  "title": "Google CMIP6",
+  "description": "This is an ESM collection for CMIP6 Zarr data residing in Pangeo's Google Storage.",
+  "extent": {
+    "spatial": {
+      "bbox": [[-180, -90, 180, 90]]
+    },
+    "temporal": {
+      "interval": [["1850-01-15T12:00:00Z", "2014-12-15T12:00:00Z"]]
+    }
+  },
+  "providers": [
+    {
+      "name": " World Climate Research Programme",
+      "roles": ["producer","licensor"],
+      "url": "https://www.wcrp-climate.org/wgcm-cmip/wgcm-cmip6"
+    },
+    {
+      "name": "The Pangeo Project",
+      "roles": ["processor"],
+      "url": "https://console.cloud.google.com/pangeo.io"
+    },
+    {
+      "name": "Google",
+      "roles": ["host"],
+      "url": "https://console.cloud.google.com/marketplace/details/noaa-public/cmip6"
+    }
+  ],
+  "license": "proprietary",
+  "links": [
+    {
+      "href": "https://pcmdi.llnl.gov/CMIP6/TermsOfUse/TermsOfUse6-1.html",
+      "type": "text/html",
+      "rel": "license",
+      "title": "CMIP6: Terms of Use"
+    }
+  ],
+  "assets": {
```

Comment:
Could this key be something else to avoid collision with the existing `asset` extension that modifies collections? Or follow the `extensionName:key` pattern? As is it's impossible for a collection to implement both extensions

Reply:
The existing asset definition extension has been renamed, partly because we want collection and item asset behave the same way.

Reply:
I see, I got confused by the `dev` vs. `master` thing again.